x/emissions/keeper/genesis.go

```diff
@@ -964,6 +964,29 @@ func (k *Keeper) InitGenesis(ctx context.Context, data *types.GenesisState) erro
 		}
 	}
 
+	// LastMedianInferences
+	if len(data.LastMedianInferences) != 0 {
+		for _, topicIdAndDec := range data.LastMedianInferences {
```

Comment:
```suggestion
	for _, topicIdAndDec := range data.LastMedianInferences {
```
This may be simplified I guess

Reply:
Ty! And actually there are a bunch of these!

Reply:
Done in all places where it happened in `InitGenesis`. Which was basically all places lol :P